src/main/java/org/opentripplanner/api/resource/Routers.java

```diff
@@ -213,13 +215,17 @@ public Response postGraphOverWire (
         try {
             graph = Graph.load(is, level);
             GraphService graphService = otpServer.getGraphService();
+            ObjectMapper mapper = new ObjectMapper();
+
+            JsonNode routerJsonConf = mapper.readTree(graph.routerConfig);
+            graphService.registerGraph(routerId, new MemoryGraphSource(routerId, graph, routerJsonConf));
```

Comment:
Makes sense what you write. Though this is beyond my understanding of this project/ beyond my Java skills.